Rankings.html

```diff
@@ -24,3 +24,24 @@
 		<footer></footer>
 	</body>
 </html>
+
```

Comment:
If you are adding the CSS in the HTML file itself, you need to put it in `<style>` tags inside the `<head>` section.